examples/js/e-commerce/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "example-instantsearch-ecommerce",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
This was likely caused by this `feat` in 7ba7e633e1dd582557217196e0009af0582aaeca but there was not change aside from the README in this example.